util/ros1msg/transform_test.go

```diff
@@ -114,6 +114,104 @@ string frame_id
 	}
 }
 
+func TestParseROS1MessageDefinition2(t *testing.T) {
```

Comment:
let's dump a bunch of schemas from public data into a test data directory and then test against them all.